### PR TITLE
Add support for markdown

### PR DESCRIPTION
**Pull request by dotMorten**
image

Note: All the deleted coded was mostly just moved to the HtmlToView helper classes, for reuse between popupviewer and feature forms

### CODE REVIEW

File: src/Toolkit/Toolkit.Maui/Internal/HtmlToView.Maui.cs [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+
+        if (node.FontColor.HasValue)
+            el.TextColor = ConvertColor(node.FontColor.Value);
+        if (node.BackColor.HasValue)
+            el.BackgroundColor = ConvertColor(node.BackColor.Value);
+        if (node.FontSize.HasValue)
+            el.FontSize = 16d * node.FontSize.Value; // based on AGOL's default font size
+
+        if (node.IsMonospace.HasValue && node.IsMonospace.Value)
+            el.FontFamily = "Courier New";
```
Comment (mstefarov) [inline src/Toolkit/Toolkit.Maui/Internal/HtmlToView.Maui.cs:377]: If you want to match Calcite's font choice on all platforms, consider complicating this a bit:

```suggestion
        if (node.IsMonospace.HasValue && node.IsMonospace.Value)
        {
            el.FontFamily = DeviceInfo.Platform switch
            {
                DevicePlatform.Android     => "monospace",
                DevicePlatform.iOS         => "Menlo",
                DevicePlatform.MacCatalyst => "Monaco",
                _                          => "Consolas"
            };
        }
```

File: src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.Windows.cs [hunk 2]
```diff
@@ -44,10 +54,53 @@ protected override void OnApplyTemplate()
...
+        {
+            if (_textContainer is not null)
+            {
+                var text = Element?.Text ?? string.Empty;
+#if WPF
+                if (Element?.Format == FormTextFormat.Markdown)
+                {
+                    var pipeline = new Markdig.MarkdownPipelineBuilder().Build();
+                    var result = Markdig.Markdown.ToHtml(text ?? string.Empty, pipeline);
+                    _textContainer.Document = HtmlToView.ToFlowDocument(result, (s,e) => FeatureFormView.GetFeatureFormViewParent(s as DependencyObject)?.OnHyperlinkClicked(e.Uri));
```
Comment (mstefarov) [inline src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormElementView.Windows.cs:76]:
```suggestion
                    _textContainer.Document = HtmlToView.ToFlowDocument(result, (s, e) => FeatureFormView.GetFeatureFormViewParent(s as DependencyObject)?.OnHyperlinkClicked(e.Uri));
```

File: src/Toolkit/Toolkit.Maui/Esri.ArcGISRuntime.Toolkit.Maui.csproj [hunk 3]
```diff
@@ -37,6 +37,7 @@
     <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible($(TargetFramework), 'net8.0'))">
       <PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiVersion)" />
       <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiVersion)" />
+      <PackageReference Include="Markdig" Version="0.31.0" />
```
[the remark this answers is not in the document]
Comment (dotMorten) [reply src/Toolkit/Toolkit.Maui/Esri.ArcGISRuntime.Toolkit.Maui.csproj:40]: Problem with referencing latest is anyone using an older version would be forced to update to at least that version, so older is better for class libraries, unless you need a bugfix or feature. 0.31.0 is also what MAUI uses for their markdown renderers.

File: src/Toolkit/Toolkit.Maui/Internal/HtmlToView.Maui.cs (hunk 1 above)
Comment (dotMorten) [reply src/Toolkit/Toolkit.Maui/Internal/HtmlToView.Maui.cs:377]: This isn't a calcite library. "Courier New" works on all platforms and is part of the standard set of fonts any device has past, present and future. For consistency I think it's better to stick to that.

**Pull request merged**